lambdabot-hangman-plugins/src/Lambdabot/Plugin/Hangman/Hangman.hs

```diff
@@ -102,8 +103,13 @@ commandAddPhrase [] = say incorrectArgumentsForAddPhrase
 commandAddPhrase phrase =
   withMS $ \game writer -> do
     let configuration = getConfiguration game
-    writer $ addPhrase game phrase
-    say $ messagePhraseAdded configuration
+    let upperPhrase = map toUpper phrase
+    let invalidCharacters = filter (`notElem` validCharacters) upperPhrase
```

Comment:
Readability suggestion. 
```
let isValid = null  $ filter (`notElem` validCharacters) upperPhrase
case isValid of ...
```

Reply:
I had thought of listing the invalid characters, but skipped it when generating the error message.  My thought was that invalid characters probably wouldn't print properly anyway (i.e. Unicode).  I'll apply your suggestion.  It is better, for sure.